services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/SwitchRulesSpec.groovy

```diff
@@ -213,6 +216,56 @@ class SwitchRulesSpec extends BaseSpecification {
         ]
     }
 
+    @Unroll
+    @Tags(Tag.NEGATIVE)
+    def "Deleting switch rules by non-existing #data.description is possible but any rules are not deleted"() {
```

Comment:
Is it valid to say `any rules are not deleted` in English?
I would expect `no rules are deleted"`

Reply:
Yes, it's valid but it doesn't matter already.